### ISSUE TITLE
Device name changes not taking effect in the query log

### ISSUE THREAD

**Issue by emeritaacuity0u** [AdguardTeam/AdGuardDNS#816]
### Platform

macOS

### Protocol

DNS-over-TLS

### Do you use AdGuard app?

No I don't

### Issue Details

1. Change name of client
2. Check query log for client name change

10 minutes had passed since the name change, and the changes still had not taken effect.

### Expected Behavior

Query log client name changes to new name

### Actual Behavior

Client name does not change in query log.

### Screenshots

<details><summary>Screenshot 1:</summary>

<img width="1724" alt="Screenshot 2024-08-12 at 10 21 20" src="https://github.com/user-attachments/assets/cb3279dc-38da-4149-b525-aa36b4dbebca">

</details>

**Comment by Chinaski1** [comment]
The issue is not reprodueced on my side device instantly changes the name in the filter log and in the log filters after changing the name

**Comment by emeritaacuity0u** [comment]
I changed the device name again, this time the name in the query log changed in an adequate time, however the name in the filter dropdown for devices took a few minutes to change.

**Comment by Chinaski1** [comment]
I agree, this requires an additional page refresh which is not very correct behaviour in itself

Internal ID: AGDNS-2388
